templates/index.html

```diff
@@ -1497,6 +1497,14 @@ <h2>
       secondaryUrl.href = selectedTakeover.secondary_url;
       secondaryUrl.innerHTML = selectedTakeover.secondary_cta + "&nbsp;&rsaquo;";
     }
+
+    dataLayer.push({
+      event: "NonInteractiveGAEvent",
+      eventCategory: "www.ubuntu.com-impression-takeover",
+      eventAction: "from:" + window.location.href + " to:" + selectedTakeover.primary_url,
+      eventLabel: selectedTakeover.primary_cta,
```

Comment:
Maybe this one should just the takeover title and the eventAction above be simply "CTA click"?

Reply:
Makes sense to me, will update now